jujugui/static/gui/src/app/assets/css/inspector/_juju-inspector.scss

```diff
@@ -188,6 +192,26 @@
                 }
             }
         }
+
+        button,
+        .button {
+            box-sizing: border-box;
+            background-color: #fff;
+            background-image: none;
+            border: 1px solid #cdcdcd;
+            border-radius: $border-radius;
+            color: $charm-panel-orange;
+            text-align: center;
+
+            &:hover {
+                background-color: #f5f5f5;
+            }
+        }
+
+        button {
+            @extend .button;
```

Comment:
That's actually including the mixin classes, I've moved the extend to before the button definition.